app/js/services/services.js

```diff
@@ -217,6 +217,36 @@ KMCServices.factory('PlayerService', ['$http', '$modal', '$log', '$q', 'apiServi
 			}
 			return currentRefresh.promise;
 		};
+		var getLastestPlayerProductVersion = function(){
+			if (playersService.playerProductVersion === undefined) {
+				var kmc = window.parent.kmc;
+				if (kmc && kmc.vars && kmc.vars.studioV3 && kmc.vars.studioV3.playerConfVars) {
+					try {
+						playersService.playerProductVersion = JSON.parse(kmc.vars.studioV3.playerConfVars);
+					} catch (e) {
+						console.error(e);
+					}
+				}
+				playersService.playerProductVersion = angular.isObject(playersService.playerProductVersion) &&  playersService.playerProductVersion.version ? playersService.playerProductVersion.version : "";
+			}
+			return playersService.playerProductVersion;
+		};
+
+		var getBetaPlayerProductVersion = function(){
```

Comment:
Too much duplicated code. I meant to expose only one API and use the param. like `kmc.vars.studioV3.player${param}ConfVars`

Reply:
done, please see